### PR TITLE
Removed Delta suffix of TemperatureDelta units (Fixes #331).

### LINKED ISSUE #331: Remove 'Delta' suffix of TemperatureDelta units
It is redundant as the delta semantics is already encoded in the quantity, so you will never mistake temperature units from temperature delta units.

- [ ] Add a copy of current [TemperatureDelta.json](https://github.com/angularsen/UnitsNet/blob/master/UnitsNet/UnitDefinitions/TemperatureDelta.json) units without the 'Delta' suffix
- [ ] Mark the old units in [TemperatureDelta.json](https://github.com/angularsen/UnitsNet/blob/master/UnitsNet/UnitDefinitions/TemperatureDelta.json) as obsolete (see Tablespoon in Volume.json as reference https://github.com/angularsen/UnitsNet/blob/b6e12fcf6a85b9cb86693615475792318466d27e/UnitsNet/UnitDefinitions/Volume.json#L255

Refs #180

### PR DESCRIPTION
**Pull request by dutts**
Fixed generate-code.bat to allow spaces in script path.

### CODE REVIEW

File: UnitsNet/UnitDefinitions/TemperatureDelta.json [hunk 1]
```diff
@@ -3,9 +3,106 @@
   "BaseUnit": "KelvinDelta",
```
Comment (angularsen) [inline UnitsNet/UnitDefinitions/TemperatureDelta.json]: Change this to Kelvin.

File: UnitsNet/UnitDefinitions/TemperatureDelta.json [hunk 2]
```diff
@@ -3,9 +3,106 @@
...
   "Units": [
+    {
+      "SingularName": "Kelvin",
+      "PluralName": "Kelvins",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "▲K" ]
```
Comment (angularsen) [inline UnitsNet/UnitDefinitions/TemperatureDelta.json]: Change symbols to ∆

See discussions:
https://github.com/angularsen/UnitsNet/issues/327
https://github.com/angularsen/UnitsNet/issues/328

File: UnitsNet/UnitDefinitions/TemperatureDelta.json [hunk 3]
```diff
@@ -90,6 +193,7 @@
     {
       "SingularName": "DegreeRoemerDelta",
       "PluralName": "DegreesRoemerDelta",
+      "ObsoleteText": "Deprecated due to github issue #180, please use DegreeRoemer instead",
```
Comment (angularsen) [inline UnitsNet/UnitDefinitions/TemperatureDelta.json]: Nice description!

File: generate-code.bat [hunk 4]
```diff
@@ -1,3 +1,3 @@
 @echo off
 SET scriptdir=%~dp0
-powershell -ExecutionPolicy Bypass -NoProfile -File %scriptdir%UnitsNet\Scripts\GenerateUnits.ps1
+powershell -ExecutionPolicy Bypass -NoProfile -File "%scriptdir%UnitsNet\Scripts\GenerateUnits.ps1"
```
Comment (angularsen) [inline generate-code.bat]: Ah, good catch.

File: UnitsNet.Tests/CustomCode/TemperatureDeltaTests.cs [hunk 5]
```diff
@@ -52,5 +52,14 @@ public class TemperatureDeltaTests : TemperatureDeltaTestsBase
...
         protected override double KelvinsDeltaInOneKelvinDelta => 1;
+
+        protected override double DegreesCelsiusInOneKelvinDelta => 1;
+        protected override double DegreesDelisleInOneKelvinDelta => -1.5d;
+        protected override double DegreesFahrenheitInOneKelvinDelta => 1.8;
+        protected override double DegreesNewtonInOneKelvinDelta => 0.33d;
+        protected override double DegreesRankineInOneKelvinDelta => 1.8;
+        protected override double DegreesReaumurInOneKelvinDelta => 0.8;
+        protected override double DegreesRoemerInOneKelvinDelta => 21 / 40d;
+        protected override double KelvinsInOneKelvinDelta => 1;
```
Comment (angularsen) [inline UnitsNet.Tests/CustomCode/TemperatureDeltaTests.cs]: When changing base unit to Kelvin as commented below and regenerating the code, then this test code must be updated too.

**Comment by dutts** [comment]
Will address the comments, cheers

File: UnitsNet.Tests/CustomCode/TemperatureTests.cs [hunk 6]
```diff
@@ -94,7 +95,7 @@ public void MultiplyByTemperatureDeltaEqualsTemperature(TemperatureUnit unit, in
         public void TemperatureDeltaPlusTemperatureEqualsTemperature(TemperatureUnit unit, int deltaVal, int temperatureVal, string expected)
         {
             Temperature temperature = Temperature.From(temperatureVal, unit);
-            TemperatureDelta delta = TemperatureDelta.From(deltaVal, (TemperatureDeltaUnit) unit);
+            TemperatureDelta delta = TemperatureDelta.From(deltaVal, (TemperatureDeltaUnit)Enum.Parse(typeof(TemperatureDeltaUnit), unit.ToString()));
```
Comment (angularsen) [inline UnitsNet.Tests/CustomCode/TemperatureTests.cs]: Good catch

**Comment by angularsen** [comment]
Nuget 3.84 on its way out

**Comment by dutts** [comment]
Awesome! I’ll get cracking on the other issue when I get a second.

**Pull request merged**